apps/dokploy/templates/mailserver/docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+services:
+  mailserver:
+    image: ghcr.io/docker-mailserver/docker-mailserver:latest
+    hostname: ${DMS_HOSTNAME}
```

Comment:
This template doesn't appear in the templates list

Reply:
It's my mistake, this file exists in another pull request. Will remove it.